src/install/local/install.md

```diff
@@ -0,0 +1,21 @@
+# Installation
+----
+### There are two options for installation:
+
+- __Semi-automatic Install__ - I want to install all system dependencies myself to be aware of and manage the MDAI requirements alongside my other software requirements.
```

Comment:
I almost wonder if this is better as a table. 🤔

Reply:
Hold for now, capturing in a doc ticket for better formatting review

Reply:
https://mydecisive.atlassian.net/browse/DOC-8